esphome/components/mhz19/sensor.py

```diff
@@ -45,6 +46,9 @@
                 state_class=STATE_CLASS_MEASUREMENT,
             ),
             cv.Optional(CONF_AUTOMATIC_BASELINE_CALIBRATION): cv.boolean,
+            cv.Optional(CONF_WARMUP_TIME, default="75s"): cv.templatable(
+                cv.positive_time_period_seconds
+            ),
```

Comment:
This cannot be templatable
```suggestion
            cv.Optional(CONF_WARMUP_TIME, default="75s"): cv.positive_time_period_seconds,
```